src/components/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -272,6 +276,7 @@ const AddContent = ({ isLoading }: Props) => {
           ({ name, label }) => url.includes(name) || url.includes(label),
         )?.label;
         if (newVersion) versions = [newVersion];
+        if (isEmpty(versions)) versions = ['any'];
```

Comment:
This fixes a rare bug where the UI sends an empty string instead of "any" or ["any"] for arch/version. 
I know the API corrects this, but thought it better practice to enforce from ui as well.